test/cpp/lite_interpreter_runtime/test_lite_interpreter_runtime.cpp

```diff
@@ -171,6 +173,81 @@ Traceback of TorchScript (most recent call last):
   )";
   ASSERT_THROWS_WITH_MESSAGE(mlm.forward(inputs), error_pattern);
 }
+
+TEST(RunTimeTest, ParseBytecode) {
+  // A simple example to show a simple bytecode that can be used independent of
+  // PyTorch TorchScript serialization (unpickler, etc) and operator library.
+  // It has basic control flow (if, else) and basic data orchestration (list
+  // construction). The original PyTorch program:
+
+  //  class Module(torch.nn.Module):
+  //
+  //    def __init__(self):
+  //      super().__init__()
+  //
+  //    def forward(self, x: int, h: int, xfirst: bool):
+  //      if xfirst:
+  //        return [x, h]
+  //      else:
+  //        return [h, x]
+
+  // 1. Prepare for the bytecode. In reality it can be from a customized
+  // deserializer.
+  std::vector<IValue> instructions{
+      Tup({"STOREN", 1, 4}),
+      Tup({"DROPR", 1, 0}),
+      Tup({"MOVE", 4, 0}),
+      Tup({"JF", 5, 0}),
+      Tup({"LOAD", 2, 0}),
+      Tup({"LOAD", 3, 0}),
+      Tup({"LIST_CONSTRUCT", 0, 2}),
+      Tup({"JMP", 4, 0}),
+      Tup({"LOAD", 3, 0}),
+      Tup({"LOAD", 2, 0}),
+      Tup({"LIST_CONSTRUCT", 1, 2}),
+      Tup({"STORE", 5, 0}),
+      Tup({"DROPR", 3, 0}),
+      Tup({"DROPR", 2, 0}),
+      Tup({"MOVE", 5, 0}),
+      Tup({"RET", 0, 0}),
+  };
+  std::vector<IValue> operators; // empty for this example
+  std::vector<IValue> constants; // empty for this example
+
+  std::vector<IValue> types{"List[int]", "List[int]"};
+  auto codeTable = Table(
+      {{"instructions", Tup(instructions)},
+       {"operators", Tup(operators)},
+       {"constants", Tup(constants)},
+       {"types", Tup(types)},
+       {"register_size", 5}});
+
+  // 2. Parse the function
+  std::string function_name("test_functoin");
+  auto function = std::unique_ptr<mobile::Function>(
+      new mobile::Function(c10::QualifiedName(function_name)));
+  parseInstructions(function_name, codeTable, IValue(), function.get());
```

Comment:
The original parseFunction also parses operator (with dispatcher) and FunctionSchema, which may not be needed for some systems like embedded. When we have functor to be passed with different implementations, we could unify it to one parseFunction for all components. If a table is not needed (for embedded, for example), it could be a dummy functor returning an empty table. I can have the function pointers implemented either in this PR or a follow up PR. How do you think?